src/main/java/frc/robot/subsystems/BallPath/Shooter/PIDShooterImpl.java

```diff
@@ -291,12 +267,15 @@ public boolean blocking() {
         double absoluteWheelSpeed = Math.abs(shooterEncoderReadingVelocity);
         boolean spinningDown = setPointShooterPID == 0 && absoluteWheelSpeed > threshold;
         return spinningUp || spinningDown;
-        // return false;
     }
 
     @Override
     public boolean readyToShoot(){
-        return turretReady && hoodReady && shooterPid.atSetpoint();
+        boolean shooterReady = false;
+        if(Math.abs(shooterEncoderReadingVelocity) > setPointShooterPID - 500 && Math.abs(shooterEncoderReadingVelocity) < shooterEncoderReadingVelocity + 500){
```

Comment:
This is a hacky way to apply a threshold/tolerance to the PID loop being at setpoint. The proper way is to just call `atSetpoint` as it had before, but to adjust the tolerance where it will report itself as being at setpoint:

https://github.com/FRC3161/Astrilde2022/blob/main/src/main/java/frc/robot/subsystems/BallPath/Shooter/PIDShooterImpl.java#L88